cypress/e2e/modelPlan.spec.js

```diff
@@ -98,10 +98,11 @@ describe('The Model Plan Form', () => {
         /\/models\/.{36}\/task-list\/basics\/overview/
       );
     });
-    cy.get('#ModelType-Voluntary')
-      .first()
-      .check({ force: true })
-      .should('be.checked');
+
+    cy.wait(500);
```

Comment:
I wish I didn't have to do this, but it was the only solution I could find in about 30 minutes lol.

Tried waiting for `GetModelBasics` query, but no 🎲